finality-aleph/src/network/manager/mod.rs

```diff
@@ -47,11 +50,11 @@ impl<M: Multiaddress> AuthData<M> {
 pub type Authentication<M> = (AuthData<M>, Signature);
 
 /// Data inside session, sent to validator network.
-pub type DataInSession<D> = (D, SessionId);
+pub type DataInSession<D> = NetworkDataInSession<D>;
```

Comment:
`NetworkDataInSession` should probably just be defined here (likely as `DataInSession`), with only `Version` remaining in `compatibility.rs`.

Reply:
done